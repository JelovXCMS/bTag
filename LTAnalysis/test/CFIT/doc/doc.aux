\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0 << /S /D >> }
\citation{BarlowStat}
\citation{Barlow}
\citation{Minuit}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{3}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Install}{3}{section.2}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Fit procedure}{3}{section.3}}
\citation{Barlow}
\@writefile{toc}{\contentsline {section}{\numberline {4}Estimation of the fit uncertainty}{5}{section.4}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Scale factor measurement}{5}{section.5}}
\@writefile{toc}{\contentsline {section}{\numberline {6}User method description}{5}{section.6}}
\citation{Barlow}
\@writefile{toc}{\contentsline {section}{\numberline {7}Examples}{8}{section.7}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.1}Single fit}{8}{subsection.7.1}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Result of the fit.}}{11}{figure.1}}
\newlabel{fig:result}{{1}{11}{Result of the fit}{figure.1}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces Shape comparison of the templates.}}{11}{figure.2}}
\newlabel{fig:shape}{{2}{11}{Shape comparison of the templates}{figure.2}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {3}{\ignorespaces Fractional systematic uncertainties after optimization procedure.}}{12}{figure.3}}
\newlabel{fig:sys}{{3}{12}{Fractional systematic uncertainties after optimization procedure}{figure.3}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {4}{\ignorespaces Correlation matrix.}}{12}{figure.4}}
\newlabel{fig:matrix}{{4}{12}{Correlation matrix}{figure.4}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {7.2}Scale factor measurement}{13}{subsection.7.2}}
\newlabel{sec:exampleSF}{{7.2}{13}{Scale factor measurement\relax }{subsection.7.2}{}}
\bibcite{Minuit}{1}
\bibcite{Barlow}{2}
\bibcite{BarlowStat}{3}
