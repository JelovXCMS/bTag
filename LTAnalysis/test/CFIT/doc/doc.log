This is pdfTeXk, Version 3.141592-1.40.3 (Web2C 7.5.6) (format=latex 2013.10.21)  18 JUN 2015 14:56
entering extended mode
 %&-line parsing enabled.
**doc.tex
(./doc.tex
LaTeX2e <2005/12/01>
Babel <v3.8h> and hyphenation patterns for english, usenglishmax, dumylang, noh
yphenation, arabic, basque, bulgarian, coptic, welsh, czech, slovak, german, ng
erman, danish, esperanto, spanish, catalan, galician, estonian, farsi, finnish,
 french, greek, monogreek, ancientgreek, croatian, hungarian, interlingua, ibyc
us, indonesian, icelandic, italian, latin, mongolian, dutch, norsk, polish, por
tuguese, pinyin, romanian, russian, slovenian, uppersorbian, serbian, swedish, 
turkish, ukenglish, ukrainian, loaded.
(/usr/share/texmf/tex/latex/base/article.cls
Document Class: article 2005/09/16 v1.4f Standard LaTeX document class
(/usr/share/texmf/tex/latex/base/size12.clo
File: size12.clo 2005/09/16 v1.4f Standard LaTeX file (size option)
)
\c@part=\count79
\c@section=\count80
\c@subsection=\count81
\c@subsubsection=\count82
\c@paragraph=\count83
\c@subparagraph=\count84
\c@figure=\count85
\c@table=\count86
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
(/usr/share/texmf/tex/latex/base/fontenc.sty
Package: fontenc 2005/09/27 v1.99g Standard LaTeX package

(/usr/share/texmf/tex/latex/base/t1enc.def
File: t1enc.def 2005/09/27 v1.99g Standard LaTeX file
LaTeX Font Info:    Redeclaring font encoding T1 on input line 43.
))
(/usr/share/texmf/tex/latex/base/inputenc.sty
Package: inputenc 2006/05/05 v1.1b Input encoding file
\inpenc@prehook=\toks14
\inpenc@posthook=\toks15

(/usr/share/texmf/tex/latex/base/latin1.def
File: latin1.def 2006/05/05 v1.1b Input encoding file
))
(/usr/share/texmf/tex/latex/graphics/color.sty
Package: color 2005/11/14 v1.0j Standard LaTeX Color (DPC)

(/usr/share/texmf/tex/latex/config/color.cfg
File: color.cfg 2007/01/18 v1.5 color configuration of teTeX/TeXLive
)
Package color Info: Driver file: dvips.def on input line 130.

(/usr/share/texmf/tex/latex/graphics/dvips.def
File: dvips.def 1999/02/16 v3.0i Driver-dependant file (DPC,SPQR)
)
(/usr/share/texmf/tex/latex/graphics/dvipsnam.def
File: dvipsnam.def 1999/02/16 v3.0i Driver-dependant file (DPC,SPQR)
))
(/usr/share/texmf/tex/latex/hyperref/hyperref.sty
Package: hyperref 2007/02/07 v6.75r Hypertext links for LaTeX

(/usr/share/texmf/tex/latex/graphics/keyval.sty
Package: keyval 1999/03/16 v1.13 key=value parser (DPC)
\KV@toks@=\toks16
)
\@linkdim=\dimen103
\Hy@linkcounter=\count87
\Hy@pagecounter=\count88

(/usr/share/texmf/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2007/02/07 v6.75r Hyperref: PDFDocEncoding definition (HO)
)
(/usr/share/texmf/tex/latex/config/hyperref.cfg
File: hyperref.cfg 2002/06/06 v1.2 hyperref configuration of TeXLive
)
(/usr/share/texmf/tex/latex/oberdiek/kvoptions.sty
Package: kvoptions 2006/08/22 v2.4 Connects package keyval with LaTeX options (
HO)
)
Package hyperref Info: Hyper figures OFF on input line 2288.
Package hyperref Info: Link nesting OFF on input line 2293.
Package hyperref Info: Hyper index ON on input line 2296.
Package hyperref Info: Plain pages OFF on input line 2303.
Package hyperref Info: Backreferencing OFF on input line 2308.

Implicit mode ON; LaTeX internals redefined
Package hyperref Info: Bookmarks ON on input line 2444.
(/usr/share/texmf/tex/latex/ltxmisc/url.sty
\Urlmuskip=\muskip10
Package: url 2005/06/27  ver 3.2  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 2599.
\Fld@menulength=\count89
\Field@Width=\dimen104
\Fld@charsize=\dimen105
\Choice@toks=\toks17
\Field@toks=\toks18
Package hyperref Info: Hyper figures OFF on input line 3102.
Package hyperref Info: Link nesting OFF on input line 3107.
Package hyperref Info: Hyper index ON on input line 3110.
Package hyperref Info: backreferencing OFF on input line 3117.
Package hyperref Info: Link coloring OFF on input line 3122.
\Hy@abspage=\count90
\c@Item=\count91
\c@Hfootnote=\count92
)
*hyperref using default driver hdvips*
(/usr/share/texmf/tex/latex/hyperref/hdvips.def
File: hdvips.def 2007/02/07 v6.75r Hyperref driver for dvips

(/usr/share/texmf/tex/latex/hyperref/pdfmark.def
File: pdfmark.def 2007/02/07 v6.75r Hyperref definitions for pdfmark specials
\pdf@docset=\toks19
\pdf@box=\box26
\pdf@toks=\toks20
\pdf@defaulttoks=\toks21
\Fld@listcount=\count93
))
(/usr/share/texmf/tex/latex/placeins/placeins.sty
Package: placeins 2005/04/18  v 2.2
)
(/usr/share/texmf/tex/latex/graphics/graphicx.sty
Package: graphicx 1999/02/16 v1.0f Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/share/texmf/tex/latex/graphics/graphics.sty
Package: graphics 2006/02/20 v1.0o Standard LaTeX Graphics (DPC,SPQR)

(/usr/share/texmf/tex/latex/graphics/trig.sty
Package: trig 1999/03/16 v1.09 sin cos tan (DPC)
)
(/usr/share/texmf/tex/latex/config/graphics.cfg
File: graphics.cfg 2007/01/18 v1.5 graphics configuration of teTeX/TeXLive
)
Package graphics Info: Driver file: dvips.def on input line 90.
)
\Gin@req@height=\dimen106
\Gin@req@width=\dimen107
)
Package hyperref Info: Option `colorlinks' set `true' on input line 18.

(/usr/share/texmf/tex/latex/preprint/authblk.sty
Package: authblk 2001/02/27 1.3 (PWD)
\affilsep=\skip43
\@affilsep=\skip44
\c@Maxaffil=\count94
\c@authors=\count95
\c@affil=\count96
) (./doc.aux)
\openout1 = `doc.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 27.
LaTeX Font Info:    ... okay on input line 27.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 27.
LaTeX Font Info:    ... okay on input line 27.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 27.
LaTeX Font Info:    ... okay on input line 27.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 27.
LaTeX Font Info:    ... okay on input line 27.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 27.
LaTeX Font Info:    ... okay on input line 27.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 27.
LaTeX Font Info:    ... okay on input line 27.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 27.
LaTeX Font Info:    ... okay on input line 27.
Package hyperref Info: Link coloring ON on input line 27.

(/usr/share/texmf/tex/latex/hyperref/nameref.sty
Package: nameref 2006/12/27 v2.28 Cross-referencing by name of section

(/usr/share/texmf/tex/latex/oberdiek/refcount.sty
Package: refcount 2006/02/20 v3.0 Data extraction from references (HO)
)
\c@section@level=\count97
)
LaTeX Info: Redefining \ref on input line 27.
LaTeX Info: Redefining \pageref on input line 27.
 (./doc.out) (./doc.out)
\@outlinefile=\write3
\openout3 = `doc.out'.

LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <14.4> on input line 29.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <7> on input line 29.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <5> on input line 29.
LaTeX Font Info:    Try loading font information for T1+cmtt on input line 29.

(/usr/share/texmf/tex/latex/base/t1cmtt.fd
File: t1cmtt.fd 1999/05/25 v2.5h Standard LaTeX font definitions
) [1

] (./doc.toc
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <12> on input line 8.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <8> on input line 8.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <6> on input line 8.
)
\tf@toc=\write4
\openout4 = `doc.toc'.

 [2] [3] [4]
LaTeX Font Info:    Font shape `T1/cmtt/bx/n' in size <12> not available
(Font)              Font shape `T1/cmtt/m/n' tried instead on input line 180.

Overfull \hbox (46.11014pt too wide) in paragraph at lines 179--187
\T1/cmr/m/n/12 plates are re-placed by statis-tic ([]\T1/cmtt/m/n/12 SetStatVar
iation[]\T1/cmr/m/n/12 ) or sys-tem-atic ([]\T1/cmtt/m/n/12 SetSysVariation[]\T
1/cmr/m/n/12 )
 []

[5]
Overfull \hbox (23.624pt too wide) in paragraph at lines 224--232
[]\T1/cmtt/m/n/12 void SetMorphing(OPTMORPHMODE optMorphMode=OPTMORPH_GEOMETRIC
,float
 []


Overfull \hbox (2.31679pt too wide) in paragraph at lines 224--232
[]\T1/cmtt/m/n/12 OPTMORPH_CUTOFF[]\T1/cmr/m/n/12 : $\OML/cmm/m/it/12 f \OT1/cm
r/m/n/12 = 0$ \T1/cmr/m/n/12 for bins with $\OML/cmm/m/it/12 b[]=b[] < frac$\T1
/cmr/m/n/12 , []\T1/cmtt/m/n/12 OPTMORPH_GEOMETRIC[]\T1/cmr/m/n/12 :
 []


Overfull \hbox (9.95145pt too wide) in paragraph at lines 224--232
\OML/cmm/m/it/12 f \OT1/cmr/m/n/12 = []\OML/cmm/m/it/12 =b[]$\T1/cmr/m/n/12 , [
]\T1/cmtt/m/n/12 OPTMORPH_ARITHMETIC[]\T1/cmr/m/n/12 : $\OML/cmm/m/it/12 f \OT1
/cmr/m/n/12 = (\OML/cmm/m/it/12 b[] \OT1/cmr/m/n/12 + \OML/cmm/m/it/12 b[]\OT1/
cmr/m/n/12 )\OML/cmm/m/it/12 =\OT1/cmr/m/n/12 (2\OML/cmm/m/it/12 b[]\OT1/cmr/m/
n/12 )$\T1/cmr/m/n/12 , []\T1/cmtt/m/n/12 OPTMORPH_FUNC[]\T1/cmr/m/n/12 :
 []


Overfull \hbox (3.32767pt too wide) in paragraph at lines 261--266
[]\T1/cmtt/m/n/12 void SetCovarianceMode(COVMODE covMode=COV_MAX)[] \T1/cmr/m/n
/12 --- Method se-lec-tion
 []

[6]
Overfull \hbox (6.56291pt too wide) in paragraph at lines 327--329
[]\T1/cmtt/m/n/12 void AddTemplate(std::string name,std::string nominalName)[] 
\T1/cmr/m/n/12 --- Add
 []


Overfull \hbox (0.38942pt too wide) in paragraph at lines 332--335
[]\T1/cmtt/m/n/12 void AddTemplateTag(std::string name,std::string tagName)[] \
T1/cmr/m/n/12 --- Add
 []


Overfull \hbox (0.94066pt too wide) in paragraph at lines 338--341
[]\T1/cmtt/m/n/12 void AddTemplateUntag(std::string name,std::string untagName)
[] \T1/cmr/m/n/12 ---
 []


Overfull \hbox (1.74199pt too wide) in paragraph at lines 356--358
[]\T1/cmtt/m/n/12 void SetLegendHeader(std::string name = "")[] \T1/cmr/m/n/12 
--- Set header for the leg-
 []


Overfull \hbox (35.97098pt too wide) in paragraph at lines 361--369
[]\T1/cmtt/m/n/12 void AddSys(std::string name,std::string upName,std::string d
ownName)[]
 []

[7]
Overfull \hbox (7.1655pt too wide) in paragraph at lines 372--375
\T1/cmr/m/n/12 his-togram by a sys-tem-atic vari-a-tion, []\T1/cmtt/m/n/12 name
[] \T1/cmr/m/n/12 has to be ei-ther []\T1/cmtt/m/n/12 upName[] \T1/cmr/m/n/12 o
r []\T1/cmtt/m/n/12 downName[]\T1/cmr/m/n/12 . 
 []


Overfull \hbox (27.63977pt too wide) in paragraph at lines 399--401
[]\T1/cmtt/m/n/12 float GetNTemplate(std::string templateName)[] \T1/cmr/m/n/12
 --- Get to-tal sum of weighted
 []


Overfull \hbox (0.76439pt too wide) in paragraph at lines 404--406
[]\T1/cmtt/m/n/12 float GetErrTemplate(std::string templateName)[] \T1/cmr/m/n/
12 --- Get to-tal er-ror for
 []

[8] [9]
File: pics/result.eps Graphic file (type eps)
 <pics/result.eps>
File: pics/shape.eps Graphic file (type eps)
 <pics/shape.eps>
File: pics/sys.eps Graphic file (type eps)
 <pics/sys.eps>
File: pics/matrix.eps Graphic file (type eps)
 <pics/matrix.eps>
[10] [11] [12] [13] [14] [15] [16] (./doc.aux) ) 
Here is how much of TeX's memory you used:
 3055 strings out of 256217
 41492 string characters out of 1917094
 97438 words of memory out of 1500000
 6313 multiletter control sequences out of 10000+200000
 17085 words of font info for 40 fonts, out of 1200000 for 2000
 645 hyphenation exceptions out of 8191
 33i,7n,36p,247b,434s stack positions out of 5000i,500n,6000p,200000b,15000s

Output written on doc.dvi (16 pages, 48496 bytes).
