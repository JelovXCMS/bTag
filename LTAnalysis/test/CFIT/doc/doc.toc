\contentsline {section}{\numberline {1}Introduction}{3}{section.1}
\contentsline {section}{\numberline {2}Install}{3}{section.2}
\contentsline {section}{\numberline {3}Fit procedure}{3}{section.3}
\contentsline {section}{\numberline {4}Estimation of the fit uncertainty}{5}{section.4}
\contentsline {section}{\numberline {5}Scale factor measurement}{5}{section.5}
\contentsline {section}{\numberline {6}User method description}{5}{section.6}
\contentsline {section}{\numberline {7}Examples}{8}{section.7}
\contentsline {subsection}{\numberline {7.1}Single fit}{8}{subsection.7.1}
\contentsline {subsection}{\numberline {7.2}Scale factor measurement}{13}{subsection.7.2}
