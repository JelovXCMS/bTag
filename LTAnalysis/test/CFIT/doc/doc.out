\BOOKMARK [1][-]{section.1}{Introduction}{}
\BOOKMARK [1][-]{section.2}{Install}{}
\BOOKMARK [1][-]{section.3}{Fit procedure}{}
\BOOKMARK [1][-]{section.4}{Estimation of the fit uncertainty}{}
\BOOKMARK [1][-]{section.5}{Scale factor measurement}{}
\BOOKMARK [1][-]{section.6}{User method description}{}
\BOOKMARK [1][-]{section.7}{Examples}{}
\BOOKMARK [2][-]{subsection.7.1}{Single fit}{section.7}
\BOOKMARK [2][-]{subsection.7.2}{Scale factor measurement}{section.7}
